### DISCUSSION TITLE
EJBCA Upgrade

### DISCUSSION THREAD

**Discussion by SohailGalaria** [Keyfactor/ejbca-ce#276] [Q&A]
I'm currently running EJBCA version 7.3.1 and planning to upgrade to latest version. I would like to know two things:

- Can I directly upgrade from 7.3.1 to the latest version or is there a step-by-step upgrade
- Are there any major API changes between 7.3.1 and the latest version. I have a CLM tool integrated with EJBCA and would like to know if there is some change needed

**Comment by primetomas** [comment]
Upgrade paths are described in the documentation.
https://doc.primekey.com/ejbca/ejbca-installation/upgrading-ejbca

Cheers,
Tomas
